frontend/src/employee-frontend/components/child-information/placements/NewServiceNeeds.tsx

```diff
@@ -1,29 +1,76 @@
-// SPDX-FileCopyrightText: 2017-2020 City of Espoo
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
 //
 // SPDX-License-Identifier: LGPL-2.1-or-later
 
 import { Table, Tbody, Td, Th, Thead, Tr } from 'lib-components/layout/Table'
 import { H4 } from 'lib-components/typography'
-import React from 'react'
-import { NewServiceNeed } from '../../../types/child'
+import React, { useContext, useState } from 'react'
+import { NewServiceNeed, Placement } from '../../../types/child'
 import _ from 'lodash'
-import StatusLabel from '../../common/StatusLabel'
-import { getStatusLabelByDateRange } from '../../../utils/date'
-import styled from 'styled-components'
-import { defaultMargins } from '../../../../lib-components/white-space'
 import { useTranslation } from '../../../state/i18n'
+import Toolbar from '../../common/Toolbar'
+import styled from 'styled-components'
+import InlineButton from '../../../../lib-components/atoms/buttons/InlineButton'
+import { faPlus } from 'lib-icons'
+import LocalDate from '../../../../lib-common/local-date'
+import { UUID } from '../../../types'
+import { FixedSpaceRow } from 'lib-components/layout/flex-helpers'
+import { DatePickerDeprecated } from '../../../../lib-components/molecules/DatePickerDeprecated'
+import Checkbox from '../../../../lib-components/atoms/form/Checkbox'
+import SimpleSelect from '../../../../lib-components/atoms/form/SimpleSelect'
+import { ChildContext } from '../../../state'
+import {
+  createNewServiceNeed,
+  deleteNewServiceNeed,
+  updateNewServiceNeed
+} from '../../../api/child/new-service-needs'
+import { UIContext } from '../../../state/ui'
+
+interface DropdownOption {
+  label: string
+  value: string
+}
+
 interface Props {
-  serviceNeeds: NewServiceNeed[]
+  placement: Placement
+  reload: () => void
 }
 
-function NewServiceNeeds({ serviceNeeds }: Props) {
+function NewServiceNeeds({ placement, reload }: Props) {
+  const { serviceNeeds, type: placementType } = placement
+
   const { i18n } = useTranslation()
   const t = i18n.childInformation.placements.serviceNeeds
 
+  const [creatingNew, setCreatingNew] = useState(false)
+  const [editingId, setEditingId] = useState<string | null>(null)
+
+  const { serviceNeedOptions } = useContext(ChildContext)
+  const options = serviceNeedOptions.isSuccess
+    ? serviceNeedOptions.value
+        .filter((opt) => opt.validPlacementType === placementType)
```

Comment:
You should also filter out the "default" options, they shouldn't be picked manually. Or at least that was my intention, maybe it was a bad idea.

Reply:
Hmm I think this needs more thought. If there is only one option which is also default (e.g. case club) I agree that even the whole service need section could be hidden. However, if we think about daycare placement type, my understanding was that the default option is used as a fallback when the service need is missing but we need some coefficients for e.g. invoicing. Nevertheless, the explicit selection is still missing, so we need to notify the unit supervisor that he needs to set the service need, which might end up being the same as the default or not. But yeah lets maybe discuss this in a call so we understand the defaults the same way.